data_discovery_ai/common/parameters.json

```diff
@@ -19,7 +19,7 @@
     "batch": 32,
```

Comment:
can you rename this file to make it be specific please? "parameters" of what, try to name something easy to understand without having to find the usage for context.

Reply:
file name changed to `keyword_classification_parameters.ini`